lib/routes/verify.js

```diff
@@ -22,6 +24,17 @@ module.exports = {
     }
   },
   handler: function verify(req, reply) {
-    token.verify(req.payload.token).done(reply, reply);
+    token.verify(req.payload.token).then(function(info) {
+      if (req.payload.email) {
+        logger.warn('email.returned', {
```

Comment:
Tiny naming nit: it's not obvious to me what "returned" means here.  Something like "requested" may be a little more intuitive.